yt_dlp/extractor/dailymotion.py

```diff
@@ -88,6 +93,310 @@ def _call_api(self, object_type, xid, object_fields, note, filter_extra=None):
+          thumbnailx60: thumbnailURL(size: "x60")
+          thumbnailx120: thumbnailURL(size: "x120")
+          thumbnailx240: thumbnailURL(size: "x240")
+          thumbnailx720: thumbnailURL(size: "x720")
+          audienceCount
+          aspectRatio
+          isOnAir
+          channel {
+            id
+            xid
+            name
+            displayName
+            accountType
+            __typename
+          }
+          __typename
+        }
+        __typename
+      }
+      __typename
+    }
+    channels(query: $query, first: $limit, page: $page) @include(if: $shouldIncludeChannels) {
+      metadata {
+        algorithm {
+          uuid
+          __typename
+        }
+        __typename
+      }
+      pageInfo {
+        hasNextPage
+        nextPage
+        __typename
+      }
+      totalCount
+      edges {
+        node {
+          id
+          ...CHANNEL_BASE_FRAG
+          __typename
+        }
+        __typename
+      }
+      __typename
+    }
+    playlists: collections(query: $query, first: $limit, page: $page) @include(if: $shouldIncludePlaylists) {
+      metadata {
+        algorithm {
+          uuid
+          __typename
+        }
+        __typename
+      }
+      pageInfo {
+        hasNextPage
+        nextPage
+        __typename
+      }
+      totalCount
+      edges {
+        node {
+          id
+          ...PLAYLIST_BASE_FRAG
+          __typename
+        }
+        __typename
+      }
+      __typename
+    }
+    topics(query: $query, first: $limit, page: $page) @include(if: $shouldIncludeTopics) {
+      metadata {
+        algorithm {
+          uuid
+          __typename
+        }
+        __typename
+      }
+      pageInfo {
+        hasNextPage
+        nextPage
+        __typename
+      }
+      totalCount
+      edges {
+        node {
+          id
+          ...TOPIC_BASE_FRAG
+          __typename
+        }
+        __typename
+      }
+      __typename
+    }
+    __typename
+  }
+}
+'''
+
+        if not self._HEADERS.get('Authorization'):
+            token = self._get_token(term)
+            self._HEADERS['Authorization'] = 'Bearer ' + token
+        resp = self._download_json(
+            'https://graphql.api.dailymotion.com/', None, note, data=json.dumps({
+                'operationName': 'SEARCH_QUERY',
+                'query': payload_living_horror,
+                'variables': {
+                    'limit': 20,
+                    'page': page,
+                    'query': term,
+                    'shouldIncludeChannels': False,
+                    'shouldIncludeLives': False,
+                    'shouldIncludePlaylists': False,
+                    'shouldIncludeTopics': False,
+                    'shouldIncludeVideos': True
+                }
+            }).encode(), headers=self._HEADERS)
+        obj = resp['data']['search']
+        if not obj:
+            raise ExtractorError(resp['errors'][0]['message'], expected=True)
```

Comment:
Should be non-fatal:
```suggestion
        obj = traverse_obj(resp, ('data', 'search'))
        if not obj:
            raise ExtractorError(traverse_obj(resp, ('errors', 0, 'message'))
                                 or 'Could not fetch search data', expected=True)
```